corehq/apps/users/models.py

```diff
@@ -2683,6 +2683,9 @@ class Invitation(models.Model):
     role = models.CharField(max_length=100, null=True)  # role qualified ID
     program = models.CharField(max_length=126, null=True)   # couch id of a Program
     supply_point = models.CharField(max_length=126, null=True)  # couch id of a Location
```

Comment:
> This is the 1st PR of three to rename the Invitation supply_point field to location_id.

This sounds great.  Also FYI in case it's helpful in the migration strategy, I think you can change the name in python without changing the name in SQL, by setting the `db_column` option on the model field.  Obviously it'd be preferable to keep the two in sync for the long term, just throwing that out there in case it's a useful intermediate state (so the python code wouldn't need to worry about the two names).
```python
    location_id = models.CharField(db_column="supply_point", max_length=126, null=True)
```